### DISCUSSION TITLE
Asus Laptop with GTX 965M - possible to mine or no?

### DISCUSSION THREAD

**Discussion by ZooBuZZ** [Lolliedieb/lolMiner-releases#484] [Q&A]
Hi,

I've been mining some on my desktop, but I do play a lot of games so I'm in and out of mining.  I have an old Asus laptop that sits idle the majority of the time though.  IT does have a dedicated card (nvidia gtx 965M).  I realize this is an old card, but is there no chance it would work?  When I try to run eth (or beam, or anything) it always gives some error like this: error when allocating memory for the Dag. Card disabled. Does anyone know can I use this laptop/card or is it just too old?

**Comment by Shillelagh-Hummus** [comment]
How much memory does it have?

**Reply by ZooBuZZ** [reply]
The gpu or the laptop?  I'm guessing the gpu.
https://www.nvidia.com/en-us/geforce/gaming-laptops/geforce-gtx-965m/specifications/

memory clock is 2500 mhz.  The laptop itself has 8gb of ram.
